x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/nio/SSLDriverTests.java

```diff
@@ -191,8 +196,17 @@ public void testHandshakeFailureBecauseProtocolMismatch() throws Exception {
 
         // Prior to JDK11 we still need to send a close alert
         if (serverDriver.isClosed() == false) {
-            failedCloseAlert(serverDriver, clientDriver, Arrays.asList("Received fatal alert: protocol_version",
-                "Received fatal alert: handshake_failure"));
+            if (false == inFipsJvm() && inZuluJvm && false == serverDriver.getOutboundBuffer().hasEncryptedBytesToFlush()) {
+                serverDriver.getSSLEngine().closeInbound();
+                serverDriver.getSSLEngine().closeOutbound();
+                serverDriver.close();;
+                assertTrue(serverDriver.isClosed());
+                clientDriver.close();
+                assertTrue(clientDriver.isClosed());
+            } else {
+                failedCloseAlert(serverDriver, clientDriver, Arrays.asList("Received fatal alert: protocol_version",
+                    "Received fatal alert: handshake_failure"));
+            }
```

Comment:
The test fails without these changes. But honestly I am half guessing here. The latest zulu update seems to get into a weird state in that the driver is not closed but it does not have any buffer to flush either. So it would error out if we simply call `failedCloseAlert`. The test actually passes if I just comment out this block entirely for zulu, but I decide to mimic the behaviour of `failedCloseAlert` as much as possible, ie. closing down both server and client SSL drivers. 
@tbrooks8 in case there is more to it.